.github/workflows/build.yml

```diff
@@ -19,6 +19,19 @@ jobs:
       with:
         credentials_json: '${{ secrets.GCP_CREDENTIALS }}'
 
+    # Run Flask migrations on staging environment
+    - name: Run Flask Migrations (Staging)
+      env:
+        FLASK_ENV: ${{ secrets.FLASK_ENV }}
+        DB_NAME: ${{ secrets.DB_NAME }}
+        DB_USER: ${{ secrets.DB_USER }}
+        DB_PASSWORD: ${{ secrets.DB_PASSWORD }}
+        DB_PORT: ${{ secrets.DB_PORT }}
+
+      run: |
+        pip install -r requirements.txt
```

Comment:
Do we need this step? I believe Google Cloud Function is anyhow doing this. Can we check once?

Reply:
Sure, @Satendra-SR. Let me confirm once.